src/components/ListUniqueValues.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <div class="list-unique-values">
+    <div class="list-unique-values__title">Filter values</div>
+    <input class="list-unique-values__search-box" placeholder="Search 50 items" v-model="search" />
+    <div class="list-unique-values__select-clear-all">
+      <span class="list-unique-values__select-all" @click="selectAll">Select all</span> &nbsp;
+      <span class="list-unique-values__clear-all" @click="clearAll">Clear all</span>
+    </div>
+    <div
+      class="list-unique-values__checkbox-container"
+      v-for="(value, index) in searchedOptions"
+      :key="value.value"
+    >
+      <CheckboxWidget
+        :label="`${value.value} (${value.count})`"
+        :value="isChecked[value.value]"
+        @input="toggleCheck(index)"
+      />
+    </div>
+    <div class="list-unique-values__load-all-values" v-if="loaded">
+      <div>List maybe incomplete</div>
+      <div @click="loadAllValues" class="list-unique-values__load-all-values-button">
+        Load all values
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+// This type will be imported from @/lib/dataset/helpers.ts
+type ColumnValueStat = {
+  value: any;
+  count: number;
+};
+
+import CheckboxWidget from './stepforms/widgets/Checkbox.vue';
+@Component({
+  name: 'list-unique-values',
+  components: { CheckboxWidget },
+})
+export default class ListUniqueValues extends Vue {
+  @Prop({
+    type: Array,
+    required: true,
+  })
+  options!: ColumnValueStat[];
+
+  @Prop({
+    type: Array,
+    required: true,
+  })
+  value!: boolean[];
```

Comment:
What do you think about a docstring explaing what we intend by `value` here?
If I understand correctly, it's an array, indexed the same way as options, that say if a value is checked or not?

Reply:
I still think it would be better to have a data structure saying if values are included or excluded.
By default, no value should be excluded.

I think this will prevent cases where the data table is updated with new values, which would be masked if the values are only included in a positive manner.
I'll explain:

Initial dataset:
displayedData: [1, 2, 3, 4, 5]

I decide to hide the value 2:
value: [true, false, true, true, true]
displayedData: [1, 3, 4, 5]

I change something in a step:
newData: [2, 3, 4, 5, 6]
What will be displayed ? 
using position of each value, it would be: [2, 4, 5, 6]
using positively included values, it would be: [3, 4, 5] because 6 didn't exist when we added all the values to the include filter.
Using such a data filter would force to reinitialize it at each request, to keep it in sync. It may be forgotten at some point by a dev, raising such issues.

That's why I advise using a filter like:
type: 'exclude', values: [2]
Even if the dataset changes, the intent of the user is correctly applied.
It's also more compact to store in the state :p specially when there is A LOT of unique values. An frequent use case for this would be to exclude 0 values from a numerical column. 

What are your opinion on this?